css-typed-om/Overview.bs

```diff
@@ -250,18 +250,24 @@ The <dfn method for=StylePropertyMap>append(DOMString <var>property</var>,
 
     5. Let |values to append| be the empty list.
 
-    6. For each |value| in |values|:
+    6. For each |value| in |values| if the [=algorithm to check if any given value is of a certain type=] described below exits 
+        gracefully append the returned object to |values to append|
+
+    7. Append |values to append| to the end of |entry|’s list. 
+</div>
+
+<div algorithm>
```

Comment:
I think this algorithm coerces |value| into an appropriate type, as it attempts to convert DOMStrings. Also describe how it fails (i.e. by throwing a TypeError)